### ISSUE TITLE
Epsilon std in VAE tutorial not defined

### ISSUE THREAD

**Issue by guillaume-chevalier** [keras-team/keras#8942]
Hi,

This issue is about this blog post: https://blog.keras.io/building-autoencoders-in-keras.html

Please define what `epsilon_std` is. I didn't understand the VAE example until I looked into the code, but I could have figured it out if `epsilon_std` was defined in the post as equal to 1.0 as it is in the true source code here: https://github.com/keras-team/keras/blob/master/examples/variational_autoencoder.py

To summarize, it'd be great to define `epsilon_std` in the blog post's code samples rather than to let users dig into the code to see what's that variable. I had no problems understanding the rest. Or maybe the variable should be renamed so as to not require to tell its value to understand the post?

Thanks!

**Comment by jackdahms** [comment]
I'm trying to learn about VAEs at the moment, and this confused me too, except I've looked at the code and I'm still confused. Could you explain what epsilon_std is?

**Comment by kevkid** [comment]
Even worse is the latest version of the example where `epsilon_std` doesn't exist in the code.
